src/neofs_testlib/cli/neogo/wallet.py

```diff
@@ -49,20 +50,37 @@ def init(
                 conflicts with --wallet-config flag.
             wallet_config: Target location of the wallet config file; conflicts with --wallet flag.
             account: Create a new account.
+            password: Password for the new account.
 
         Returns:
             Command's result.
         """
         assert bool(wallet) ^ bool(wallet_config), self.WALLET_SOURCE_ERROR_MSG
 
-        return self._execute(
-            "wallet init",
-            **{
-                param: param_value
-                for param, param_value in locals().items()
-                if param not in ["self"]
-            },
-        )
+        if account:
+            return self._execute(
+                "wallet init",
+                **{
+                    **{param: param_value
+                    for param, param_value in locals().items()
+                    if param not in ["self", "password"]},
+                    'options': CommandOptions(
+                        interactive_inputs=[
+                            InteractiveInput(prompt_pattern="account", input=''),
+                            InteractiveInput(prompt_pattern="password", input=password),
+                            InteractiveInput(prompt_pattern="password", input=password),
```

Comment:
is this second time something like "Confirm password?" If so, need a comment here